sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/SortAggregateExec.scala

```diff
@@ -93,7 +93,7 @@ case class SortAggregateExec(
           initialInputBufferOffset,
           resultExpressions,
           (expressions, inputSchema) =>
-            newMutableProjection(expressions, inputSchema, subexpressionEliminationEnabled),
+            MutableProjection.create(expressions, inputSchema),
```

Comment:
Seems the new code path always set `subexpressionEliminationEnabled` to false. Can we provide a `create` method that takes `subexpressionEliminationEnabled` parameter?

Reply:
But, it seems `newMutableProjection` always ignores the parameter?
https://github.com/apache/spark/pull/26615/files#diff-b9f96d092fb3fea76bcf75e016799678L468

Reply:
MutableProjection now directly gets the config from SQLConf.get.subexpressionEliminationEnabled.

Reply:
yea, I think so.

Reply:
ah i see, thanks for confirmation!